tests/test_update.py

```diff
@@ -218,12 +218,18 @@ def test_too_big_image_update(
                 f"{mender_update_binary} install /var/tmp/image-too-big.mender ; echo ret_code=$?"
             )
 
+            allowed_msgs = ["no space left on device"]
+            if bitbake_variables["MACHINE"] == "vexpress-qemu-flash":
+                # On UBI, we get this message instead because we are not permitted to schedule a
```

Comment:
Is it that we always get this message or that we _can_ get it?

If for certain we are expecting "operation not permitted" I would modify the test to expect this (and only this) for flash and the default message for the others.